<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="0" name="ADAT_Test" tests="15" failures="1" timestamp="17 Feb 2019 06:09:12 GMT" time="16.242" errors="0">
  <testcase name="SetEvironmentLevel" time="0.037" classname="SupportClasses.Environment">
    <skipped/>
  </testcase> <!-- SetEvironmentLevel -->
  <testcase name="ADAT_CreateUser" time="0.655" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_CreateUser" time="0.74" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_CreatePin" time="0.348" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_CreatePin" time="0.932" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_PostcardVelocity" time="1.309" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_PhoneVelocity" time="1.447" classname="Soap_Execution.ADAT">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;&amp;lt;rf11:advice&amp;gt;&amp;quot;
     but: was &amp;quot;com.sun.xml.internal.messaging.saaj.SOAPExceptionImpl: Bad response: (400Bad Request&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "<rf11:advice>"
     but: was "com.sun.xml.internal.messaging.saaj.SOAPExceptionImpl: Bad response: (400Bad Request"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at Soap_Execution.ADAT.ADAT_EvaluateRisk(ADAT.java:303)
at Soap_Execution.ADAT.VelocityCheck(ADAT.java:140)
at Soap_Execution.ADAT.ADAT_PhoneVelocity(ADAT.java:104)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- ADAT_PhoneVelocity -->
  <testcase name="ADAT_AddressVelocity" time="3.635" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_PhoneVelocity" time="5.316" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_PostcardVelocity" time="5.7" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_AddressVelocity" time="12.391" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_VerifyPin" time="0.752" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_VerifyPin_Invalid" time="0.753" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_VerifyPin" time="1.711" classname="Soap_Execution.ADAT"/>
  <testcase name="ADAT_VerifyPin_Invalid" time="1.702" classname="Soap_Execution.ADAT"/>
</testsuite> <!-- ADAT_Test -->
